var/spack/repos/builtin/packages/apple-clang/detection_test.yaml

```diff
@@ -4,32 +4,20 @@ paths:
       - executables:
           - "bin/clang"
           - "bin/clang++"
-        script: |
-          echo "Apple clang version 11.0.0 (clang-1100.0.33.8)"
-          echo "Target: x86_64-apple-darwin19.5.0"
-          echo "Thread model: posix"
-          echo "InstalledDir: /Library/Developer/CommandLineTools/usr/bin"
```

Comment:
I guess this is a temporary change, right?

Reply:
No, that's a change because we changed what argument we pass to `apple-clang` to detect the version. `apple-clang -dumpversion` outputs only the version string itself, rather than all of this information that we don't need for detection.

Reply:
How do we distinguish between Apple Clang and LLVM Clang then?

Reply:
Ugh yep that's annoying. Fixed.